mongorestore/filepath_test.go

```diff
@@ -122,6 +170,8 @@ func TestCreateIntentsForDB(t *testing.T) {
 	//   db1/c2.bson
 	//   db1/c3.bson
 	//   db1/c3.metadata.json
+	//   db1/c4.bson
+	//   db1/c4.metadata.json
```

Comment:
What is c4 testing?

Reply:
I added it because of [a concern I had here](https://github.com/mongodb/mongo-tools/pull/233#discussion_r416168450). In short, it's to give us more meaningful results for testing --nsInclude and --nsExclude.

I was worried that since c1 is the only collection with docs in it, there might be a case where `--nsInclude db1.c1` could include more than it should and we wouldn't know since c2 and c3 have 0 docs. Same case with `--nsExclude db1.c1` but with opposite results. So I added c4 as a second collection with a non-zero number of docs.

I have to account for c4 in filepath_test.go since it uses the same `testdata/testdirs/db1` directory as mongorestore_test.go

Reply:
Ah I see, good solution!